content/docs/reference/autocert.mdx

```diff
@@ -15,47 +17,42 @@ import TabItem from '@theme/TabItem';
+- Pomerium Enterprise users should note that Autocert cannot be configured in the [Enterprise Console](/docs/enterprise#enterprise-console) or the Pomerium Enterprise [configuration file](/docs/enterprise/configure). Autocert is a bootstrap setting, meaning it must be configured in Pomerium Core (either in a configuration file or as an environment variable) before running Pomerium Enterprise.
 
-You must use [Autocert Directory](/docs/reference/autocert) with Autocert to store and share certificate data between services. (Note that Autocert also provides [OCSP stapling](https://en.wikipedia.org/wiki/OCSP_stapling).)
+- Kubernetes users should refer to the [Cert-Manager integration](/docs/k8s/ingress#cert-manager-integration) section for more information on configuring automated certificate management in the Pomerium Ingress Controller.
 
-The Autocert setting can be useful in situations where you may not have Pomerium behind a TLS terminating ingress or proxy that is already handling your public certificates on your behalf.
+- Pomerium Zero users should note that Pomerium automatically manages certificates for [starter domains](/docs/concepts/clusters#starter-domain) and [custom domains](/docs/capabilities/custom-domains).
 
-Autocert will incorporate certificates available in the system trust store and certificates set manually in the Pomerium configuration, and they will take precedence over generated certificates when applicable to configured routes.
+:::
+
+## Autocert {#autocert}
 
-Autocert will attempt `HTTP-01`and `TLS-ALPN-01` challenges. It does not support `DNS-01` challenges, required to generate wildcard certificates.
+The **Autocert** setting enables Pomerium to automatically retrieve, manage, and renew public-facing TLS certificates from [Let's Encrypt][letsencrypt] to secure Pomerium-managed route domains and the authenticate service (Autocert is not supported with the [Hosted Authenticate Service](/docs/capabilities/hosted-authenticate-service)).
 
-Kubernetes users should not use Autocert. See the [cert-manager's guide](https://cert-manager.io/docs/tutorials/acme/pomerium-ingress/) for more information.
+Autocert can incorporate also certificates available in the system trust store and certificates set manually in the Pomerium configuration. These certificates will take precedence over generated certificates when applicable to configured routes.
 
-:::warning
+Before Autocert can provision certificates, it must complete either the [HTTP-01](https://letsencrypt.org/docs/challenge-types/#http-01-challenge) or [TLS-ALPN-01](https://letsencrypt.org/docs/challenge-types/#tls-alpn-01) challenge. The HTTP-01 challenge requires internet access to port 80; the TLS-ALPN-01 challenge requires internet access to port 443. Autocert does not support `DNS-01` challenges, which is required to generate wildcard certificates.
```

Comment:
I think this could be clearer— "before" feels like the wrong word to describe this process. Maybe something more like this? "When requesting or renewing a certificate, Pomerium performs either the HTTP-01 or TLS-ALPN-01 challenge. The DNS-01 challenge is not supported."

The part about "requires internet access to port 80" also feels a little muddled. The important points are:
- using the HTTP-01 challenge requires setting up an HTTP redirect listener on port 80 — we can link to /docs/reference/http-redirect-address here
- using the TLS-ALPN-01 challenge requires that the Address setting uses port 443

Reply:
Ah OK, I wasn't sure if the "internet accessible" detail was important here, or if it just meant an available, open port. Seems like they're saying the same thing. I appreciate the clarification. I'll update.